Resources/Prototypes/_White/Entities/Objects/Misc/subdermal_implants.yml

```
@@ -25,3 +25,15 @@
+
+- type: entity
+  parent: BaseSubdermalImplant
+  id: MindSlaveImplant
+  name: mindslave implant
+  description: Make someone a proper doll for your use.
+  noSpawn: true
+  components:
+  - type: SubdermalImplant
+  - type: Tag
+    tags:
+    - MindSlave
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)